sale_timesheet_budget/models/project_project.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2022-2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _lt, api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    budget_ids = fields.One2many(
+        comodel_name="project.project.budget",
+        inverse_name="project_id",
+        string="Budgets",
+        copy=False,
+    )
+    budget_amount = fields.Float(compute="_compute_budget_amount")
+
+    @api.depends("budget_ids")
+    def _compute_budget_amount(self):
+        for item in self:
+            item.budget_amount = sum(item.mapped("budget_ids.amount"))
+
+    def action_profitability_budget_item(self):
+        action = self.env["ir.actions.act_window"]._for_xml_id(
+            "sale_timesheet_budget.action_project_project_budget"
+        )
+        action["domain"] = [("project_id", "=", self.id)]
+        return action
+
+    def _get_profitability_labels(self):
+        res = super()._get_profitability_labels()
+        res["budgets"] = (_lt("Budgets"),)
```

Comment:
Must be string, not tuple
```suggestion
        res["budgets"] = _lt("Budgets")
```